R/sun-methods.R

```diff
@@ -56,7 +84,10 @@ setMethod("crepuscule",
               res
           })
 
-
+#' @description Method for matrix objects. 
+#' @param crs A "CRS" object representing the coordinate reference system.
+#' Default is sf::st_crs(4326).
```

Comment:
put sf::st_crs(4326) in back ticks `sf::st_crs(4326)`. I would also mention this is WSG 84